server/sanitisedError.ts

```diff
@@ -1,18 +1,18 @@
 import type { ResponseError } from 'superagent'
 
-export interface SanitisedError extends Error {
+export interface SanitisedError<Data = unknown> extends Error {
```

Comment:
Using TS generics here so we can pass through a type for `data` (and with `unknown` as a default). Means we get TS support for the error code values in the error response.